src/Form/EntryForm.php

```diff
@@ -51,13 +59,14 @@ public function getFormID() {
    */
   public function buildForm(array $form, FormStateInterface $form_state, AccountInterface $user = null) {
     // Check flood tables.
-    //@TODO Reimplement Flood Controls.
+
```

Comment:
_tfa_hit_flood hook needs an argument , through which we can access getContext() function.
At this point, I don`t know where will be getContext() function will be moved (presently ,it is in Tfa class , which is deprecated).